dev/TeachingTip/TeachingTip.cpp

```diff
@@ -15,6 +15,7 @@ TeachingTip::TeachingTip()
     __RP_Marker_ClassById(RuntimeProfiler::ProfId_TeachingTip);
     SetDefaultStyleKey(this);
     EnsureProperties();
+    Unloaded({ this, &TeachingTip::ClosePopupOnUnloadEvent });
```

Comment:
>Unloaded({ this, &TeachingTip::ClosePopupOnUnloadEvent }); [](start = 4, length = 58)

I'm slightly concerned about how this will appear to behave to the developer.  Teaching tips can be declared anywhere and I'm not positive under which circumstances the tip will be loaded (and thus unloaded).  I did some experiments placing the teaching tip in different resource dictionaries and it behaves as I would expect, but I wonder if we haven't thought about all the scenarios.